ForceGamaAttack/src/player/Player.java

```diff
@@ -1,20 +1,65 @@
 package player;
 
-import jplay.Sprite;	
+import constants.Constants;
+import entities.BulletManager;
+import jplay.Keyboard;
+import jplay.Sprite;
+import jplay.Window;	
 
 public class Player extends Sprite{
 	private Player player;
+	private int sentidoAnterior;
+	private int sentido;
+	private boolean isDead = false;
 	
-	public Player(int x, int y) {
+	public static BulletManager bulletmanager;
+	public String bulletimg = "src/graphics/img/bullet_player.png";
+	
+	public Player(double x, double  y) {
 		super("src/graphics/img/spaceship.png", 10);
 		this.x = x;
 		this.y = y;
+		sentido = Constants.RIGHT;
+        sentidoAnterior = Constants.RIGHT;
 	}
 	
-	public void move() {
+	public void move(Keyboard teclado) {
 		
-		player.moveY(10.0);
-		player.moveX(10.0);
-	}
+		//player.moveY(10.0);
+		//player.moveX(10.0);
+		if ( teclado.keyDown(Keyboard.LEFT_KEY) && this.x > 1 ){
+				if (sentido != Constants.LEFT){   
+					sentidoAnterior = Constants.LEFT;
+					sentido = Constants.LEFT;
```

Comment:
Vamos evitar escrever código em português. Havíamos definido como padrão o código em inglês.

Reply:
I'm sorry, I forgot. But I corrected it already.